.github/pull.yml

```diff
@@ -0,0 +1,14 @@
+version: "1"
```

Comment:
what is this file used for? The title and description of the PR indicates it fixes a unit test, but this file seems unrelated

Reply:
Yes, totally not abapgit.. will remove it